docs/orchestrating-elastic-stack-applications/stack-config-policy.asciidoc

```diff
@@ -0,0 +1,237 @@
+        retention:
+          expire_after: "7d"
+          min_count: 1
+          max_count: 20
+----
+
+Another example of configuring role mapping, ingest pipeline, ILM and index templates:
+
+[source,yaml,subs="attributes,+macros"]
+----
+apiVersion: stackconfigpolicy.k8s.elastic.co/v1alpha1
+kind: StackConfigPolicy
+metadata:
+  name: test-stack-config-policy
+spec:
+  elasticsearch:
+    securityRoleMappings:
+      everyone-kibana:
+        enabled: true
+        metadata:
+          _foo: something
+          uuid: b9a59ba9-6b92-4be2-bb8d-02bb270cb3a7
+        roles:
+        - kibana_user
+        rules:
+          field:
+            username: '*'
+    ingestPipelines:
+      test-pipeline:
+        description: "optional description"
+        processors:
+        - set:
+            field: my-keyword-field
+            value: foo
+      test-2-pipeline:
+        description: "optional description"
+        processors:
+        - set:
+            field: my-keyword-field
+            value: foo
+    indexLifecyclePolicies:
+      test-ilm:
+        phases:
+          delete:
+            actions:
+              delete: {}
+            min_age: 30d
+          warm:
+            actions:
+              forcemerge:
+                max_num_segments: 1
+            min_age: 10d
+    indexTemplates:
+      componentTemplates:
+        test-component-template:
+          template:
+            mappings:
+              properties:
+                '@timestamp':
+                  type: date
+        test-runtime-component-template-test:
+          template:
+            mappings:
+              runtime:
+                day_of_week:
+                  type: keyword
+      composableIndexTemplates:
+        test-template:
+          composed_of:
+          - test-component-template
+          - test-runtime-component-template-test
+          index_patterns:
+          - test*
+          - bar*
+          priority: 500
+          template:
+            aliases:
+              mydata: {}
+            mappings:
+              _source:
+                enabled: true
+              properties:
+                created_at:
+                  format: EEE MMM dd HH:mm:ss Z yyyy
+                  type: date
+                host_name:
+                  type: keyword
+            settings:
+              number_of_shards: 1
+          version: 1
+----
+
+
+[float]
+[id="{p}-{page_id}-monitoring"]
+== Monitor Elastic Stack configuration policies
+
+In addition to the logs generated by the operator, a config policy status is maintained in the `StackConfigPolicy` resource. This status summarizes the health of the configuration policy through a phase and the number of resources for which the policy could be applied.
+
+[source,sh]
+----
+kubectl get stackconfigpolicy
+----
+
+[source,sh]
+----
+NAME                           READY   PHASE   AGE
+test-stack-config-policy       1/1     Ready   1m42s
+test-err-stack-config-policy   0/1     Error   1m42s
```

Comment:
Should these be combined into the same `[source,sh]`?

Reply:
I don't have a strong opinion, but in most pages of the doc, we keep the two separate (e.g.: [quickstart](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-deploy-elasticsearch.html#k8s-elasticsearch-monitor-cluster-health), [autoscaling](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-autoscaling.html#k8s-monitoring), [commons-problems](https://www.elastic.co/guide/en/cloud-on-k8s/current/k8s-common-problems.html)).

Reply:
No problem then @thbkrkr .  Just from reading it, I thought they should be combined as one is the command, and the other is the output of the same command, but I don't have a strong opinion either, tbh.